### PR TITLE
jf-gsm: Update all the blobs

### PR DESCRIPTION
**Pull request by zwliew**

### CODE REVIEW

**Comment by zwliew** [comment]
Not sure whether this is a good idea, but I've been using these blobs on my jfltexx for a while and they work fine.

**Comment by zwliew** [comment]
@invisiblek @javelinanddart @dcd

**Comment by dcd** [comment]
go for it

**Comment by javelinanddart** [comment]
It can't hurt, esp if it seems they are working well

File: jf-gsm-common/jf-gsm-common-vendor-blobs.mk [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 # This file is generated by device/samsung/jf-gsm-common/setup-makefiles.sh
 
-## jf-gsm-common/common-proprietary-files.txt
+## jfltexx/common-proprietary-files.txt
```
Comment (javelinanddart) [inline jf-gsm-common/jf-gsm-common-vendor-blobs.mk:17]: Although this should be changed back....

**Pull request closed without merging**